brunch-config.js

```diff
@@ -31,8 +31,7 @@ exports.config = {
   plugins: {
     babel: {
       // Do not use ES6 compiler in vendor code
-      ignore: [/^(web\/static\/vendor)/],
-      loose: "all"
```

Comment:
We could also say we don't  support IE8 and people can chase down this solution if they want it. @ericmj and others would be fine as long as he doesn't upgrade his own babel stuff.